cirq-core/requirements.txt

```diff
@@ -5,7 +5,7 @@ backports.cached_property~=1.0.1; python_version < '3.8'
 
 duet~=0.2.8
 matplotlib~=3.0
-networkx~=2.4
+networkx>=2.4 # can't do networkx>=3.0 until https://github.com/rigetti/pyquil/issues/1553 is fixed
```

Comment:
honestly I don't think this is worth mentioning. Until there is a networkx 3 feature that we actually want to use it's a _good_ thing that we support both :)